docs/api/3.0-migration.md

```diff
@@ -215,3 +215,39 @@ Due to a bug where `originalUrl` was being assigned not as the root url, but the
 * Parameter `originalUrl` from `resolveWrappers` has been correctly renamed to `previousUrl`: `resolveWrappers(ad, wrapperDepth, previousUrl)`
 * `resolveAds` param `originalUrl` in `options` object has been renamed to `previousUrl`. It also can be passed `url` into `options`. For example: `resolveAds(ads = [], { url, previousUrl })`
 * `options` parameter in `VASTParser` `parse` function now expects `previousUrl` instead of `originalUrl`. It also can be passed `url` into `options`.
+
+#### Improvement for vast parser to trigger warnings event logs
```

Comment:
this isn't actually a breaking change, because it is in a util file not exported in the module